environments/prod/kustomization.yaml

```
@@ -50,6 +50,12 @@ patches:
       - op: replace
         path: /spec/backups/pgbackrest/repos/0/volume/volumeClaimSpec/resources/requests/storage
         value: 2048Gi
+      - op: replace
+        path: /spec/backups/pgbackrest/global/repo1-retention-full-type
+        value: time # Switch retention to be based on number of days. 
+      - op: replace
+        path: /spec/backups/pgbackrest/global/repo1-retention-full
+        value: 14 # Patch to only store the last 14 days of backups.
```

This needs to be a string..

```suggestion
        value: '14' # Patch to only store the last 14 days of backups.
```

And I wonder if we should consider setting this even lower.. perhaps 7 days. We do have the velero backups, afterall.